doc/charm++/manual.rst

```diff
@@ -1923,7 +1923,9 @@ instead use a for loop:
 Note that ``int iter;`` is declared in the chare’s class definition and
 not in the ``.ci`` file. This is necessary because the Charm++ interface
 translator does not fully parse the declarations in the ``for`` loop
-header, because of the inherent complexities of C++.
+header, because of the inherent complexities of C++. Also, ``break`` and
+``continue`` statements are not supported within SDAG loops, either
+within ``serial`` blocks or otherwise.
```

Comment:
Within serial blocks, users are free to write whatever valid C++ they want, including continue and break statements. We just want to clarify that there is no breaking or continuing from an SDAG loop.